src/components/Send.jsx

```diff
@@ -92,65 +184,88 @@ export default function Payment({ currentWallet }) {
-        <rb.Form.Control.Feedback type="invalid">Please provide a receiving address.</rb.Form.Control.Feedback>
-      </rb.Form.Group>
-      <rb.Form.Group className="mb-3" controlId="account">
-        <rb.Form.Label>Account</rb.Form.Label>
-        <rb.Form.Select
-          defaultValue={account}
-          onChange={(e) => setAccount(parseInt(e.target.value, 10))}
-          style={{ maxWidth: '21ch' }}
-          required
-        >
-          {ACCOUNTS.map((val) => (
-            <option key={val} value={val}>
-              Account {val}
-            </option>
-          ))}
-        </rb.Form.Select>
-      </rb.Form.Group>
-      <rb.Form.Group className="mb-3" controlId="amount">
-        <rb.Form.Label>Amount in Sats</rb.Form.Label>
-        <rb.Form.Control name="amount" type="number" min={1} defaultValue={0} required style={{ maxWidth: '21ch' }} />
-        <rb.Form.Control.Feedback type="invalid">Please provide a valid amount.</rb.Form.Control.Feedback>
-      </rb.Form.Group>
-      <rb.Form.Group className="mb-3" controlId="isCoinjoin">
-        <rb.Form.Check
-          type="switch"
-          label="As coinjoin"
-          value={true}
-          onChange={(e) => setIsCoinjoin(e.target.checked)}
-        />
-      </rb.Form.Group>
-      {isCoinjoin === true && (
-        <rb.Form.Group className="mb-3" controlId="counterparties">
-          <rb.Form.Label>Number of counterparties</rb.Form.Label>
-          <rb.Form.Control
-            name="counterparties"
-            type="number"
-            min={0}
-            defaultValue={3}
-            style={{ width: '10ch' }}
-            required
-          />
-          <rb.Form.Control.Feedback type="invalid">Please set the counterparties.</rb.Form.Control.Feedback>
-        </rb.Form.Group>
+    <>
+      {!walletInfo ? (
+        <rb.Row className="justify-content-center">
+          <rb.Col className="flex-grow-0">
+            <div className="d-flex justify-content-center align-items-center">
+              <rb.Spinner as="span" animation="border" size="sm" role="status" aria-hidden="true" className="me-2" />
+              Loading
+            </div>
+          </rb.Col>
+        </rb.Row>
+      ) : (
+        <rb.Row className="send justify-content-center">
+          <rb.Col md={10} lg={8} xl={6}>
+            <PageTitle
+              title="Send bitcoin"
+              subtitle="You can mix your bitcoin before sending them for better privacy." // Todo: Copy.
+            />
+            {alert && <rb.Alert variant={alert.variant}>{alert.message}</rb.Alert>}
+            <rb.Form onSubmit={onSubmit} validated={validated} noValidate>
+              <rb.Form.Group className="mb-4" controlId="destination">
+                <rb.Form.Label>Recipient</rb.Form.Label>
+                <rb.Form.Control
+                  name="destination"
+                  placeholder="Enter address..."
+                  defaultValue=""
+                  className="number"
```

Comment:
`className` of `number` for input "address" not sounding quite optimal here.
Although I like the looks of it! : -)

Reply:
Fair point! :smile